CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 #### Breaking Changes
 
+#### Improvements
+
+#### Fixes
+
+## 3.0.0
```

Comment:
```suggestion
## 3.0b1
```

Reply:
shouldn't it be 3.0.0-b1?

Reply:
Also, I will have to remove the 3.0.0 tag from the commit and we can do it via a normal github release.

Reply:
No, I think beta is good.
Yes, you need to change the commit message as well. Then I will tag it and publish the release

Reply:
so `3.0.0-beta.1`?

Reply:
I'm just trying to follow https://semver.org/

Reply:
Looks good